src/phantom.js

```diff
@@ -219,12 +224,32 @@ export default class Phantom {
      */
     exit() {
         clearInterval(this.heartBeatId);
-        this.execute('phantom', 'invokeMethod', ['exit']);
+        return this.execute('phantom', 'invokeMethod', ['exit']);
+    }
+
+    /**
+     * Clean up and force kill this process
+     */
+    kill(errmsg = 'Phantom process was killed') {
+        this.rejectCommands(errmsg);
+        this.process.kill('SIGKILL');
+        return Promise.reject(errmsg);
```

Comment:
Just in case a user would want to wait for the exit method to finish 'synchronously'